### PR TITLE
bundles: addition of `cleancssurl` filter

### PR DESCRIPTION
**Pull request by drjova**
Signed-off-by: Harris Tzovanakis <me@drjova.com>

### CODE REVIEW

File: invenio_communities/bundles.py [hunk 1]
```diff
@@ -42,7 +42,7 @@
 
 css = NpmBundle(
     'scss/invenio_communities/communities.scss',
-    filters='scss, cleancss',
+    filters='scss,cleancssurl',
```
Comment (jmartinm) [inline invenio_communities/bundles.py]: Is this needed after https://github.com/miracle2k/webassets/pull/486 was merged? Or just using updated `webassets` is enough?

Comment (drjova) [inline invenio_communities/bundles.py]: Using updated ``webassets`` should be enough :D

Comment (lnielsen) [inline invenio_communities/bundles.py]: yep webassets is enough **once it is released**

**Comment by slint** [comment]
Closed since we're moving away from webassets

**Pull request closed without merging**